README.md

```diff
@@ -34,14 +34,28 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+-Semantic HTML is what allows a browser and developer to present code. It helps people understand what elements they are looking at and helps browsers understand how to show those elements.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+- Block is displayed vertically whereas inline in displayed horizonatally. 
+- Inline allows for a width and height to be set while block does not.
+
 3. What are the 4 areas of the box model?
 
+- Content
+- Border
+- Padding
+- Margin
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+- Y axis
```

Comment:
You're better off choosing between 'main axis' and 'cross axis' hear. Which one would it be in that case?